### DISCUSSION TITLE
Can't GET helm chart from oci, 403 Forbidden

### DISCUSSION THREAD

**Discussion by chrisatcho** [actions/actions-runner-controller#3407] [Questions]
Hello,
I'm trying to do the [quickstart ](https://docs.github.com/en/actions/hosting-your-own-runners/managing-self-hosted-runners-with-actions-runner-controller/quickstart-for-actions-runner-controller) and can't pull the helm chart from oci://ghcr.io/actions/actions-runner-controller-charts
Am I missing something super obvious?

Steps I take:
1. `minkube start`
2. `helm install arc --namespace "test" --create-namespace oci://ghcr.io/actions/actions-runner-controller-charts/gha-runner-scale-set-controller`

Error I receive:
`Error: INSTALLATION FAILED: GET "https://ghcr.io/v2/actions/actions-runner-controller-charts/gha-runner-scale-set-controller/tags/list": unable to retrieve credentials`

Thanks!

**Comment by DavidPriceNBS** [comment]
Did you manage to resolve this? I'm unable to install this locally, through ArgoCD, unable to pull it as a Dockerfile, just a 403 constantly everywheere

**Reply by DavidPriceNBS** [reply]
If anyone experiences the same issue - for ArgoCD at least - you just drop the OCI part like so

```
repoURL: ghcr.io/actions/actions-runner-controller-charts
targetRevision: "0.9.3"
chart: gha-runner-scale-set-controller
```
